tests/js/router/index.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+require("./router-nesting-tests");
+require("./router-params-tests");
+require("./router-static-tests");
+require("./router-wildcard-tests");
+
+// TODO:  This test causes spurious "assertion outside test context" in other tests if it's run at the same time.  Discuss with Antranig.
+//require("./router-static-failure-tests");
```

Comment:
Shouldn't we try to run it somehow all the same?
Oddly, it seems that each of these test files are trying to construct the test environment synchronously on the spot rather than queuing it properly by using the fluid.test.runTests driver - http://docs.fluidproject.org/infusion/development/IoCTestingFramework.html#-modules-tests-and-sequence-